src/fsharp/FSharp.Core/array.fsi

```diff
@@ -8,6 +8,10 @@ namespace Microsoft.FSharp.Collections
     open System.Collections.Generic
 
     /// <summary>Basic operations on arrays.</summary>
+    ///
+    /// <remarks>
+    ///  See also <a href="https://docs.microsoft.com/dotnet/fsharp/language-reference/arrays">F# Language Guide - Arrays</a>.
```

Comment:
@cartermp The FSHarp.Core API docs are vastly improved if they can cross-reference back to https://docs.microsoft.com/dotnet/fsharp/language-reference topics, e.g. for sequences, async, lists, arrays and so on.

What's the right URL we should use?  Just direct linking to the doc section seems to give the best user experience.   Setting up aka.ms links seems kind of wrong for an open source component but if you think we should do it we can.